crates/router/src/types/payment_methods.rs

```diff
@@ -102,6 +91,14 @@ impl VaultingInterface for VaultRetrieve {
     }
 }
 
+#[cfg(all(feature = "v2", feature = "payment_methods_v2"))]
+#[async_trait::async_trait]
+impl VaultingInterface for VaultDelete {
+    fn get_vaulting_request_url() -> &'static str {
+        consts::VAULT_RETRIEVE_REQUEST_URL
+    }
+}
+
```

Comment:
Shouldn't this return `VAULT_DELETE_REQUEST_URL`?

Reply:
My bad, fixing this.